contracts/ethereum/tools/lib/hatchoutContractFactory.ts

```diff
@@ -0,0 +1,47 @@
+import ContractFactory from "./factories/contractFactory";
+import VvispStateSupplier from "./suppliers/vvispStateSupplier";
+import {GhostAuctionContract, GhostFactoryContract} from "./types";
+
+export default class HatchoutContractFactory {
+  private contractFactory: ContractFactory;
+  private vvispStateSupplier: VvispStateSupplier;
+
+  constructor(privateKey?: string, endpoint?: string){
+    this.contractFactory = new ContractFactory(privateKey, endpoint);
+    this.vvispStateSupplier = new VvispStateSupplier();
+  }
+
+  public createDefaultGhostFactoryContract(): GhostFactoryContract {
+    return this.contractFactory
+      .createGhostContract(
+        this.vvispStateSupplier.loadContractAddress(
+          "GhostFactory",
+        )
+      );
+  }
+
+  public createGhostAuctionContractFromVvispState(): GhostAuctionContract {
```

Comment:
Why are the names of `createDefaultGhostFactoryContract` and `createGhostAuctionContractFromVvispstate` different?

Reply:
renamed it!